### PR TITLE
FOSSC-83 libnice

### PR DESCRIPTION
**Pull request by dierberg**

### CODE REVIEW

File: analysed-packages/libnice/version-0.1.18/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+http://nice.freedesktop.org/releases/libnice-0.1.18.tar.gz
+
+## Package URL (purl)
+
+pkg:generic/libnice@0.1.18?download_url=http://nice.freedesktop.org/releases/libnice-0.1.18.tar.gz
+
+## Creator
+
+Sascha Dierberg <dierberg@dresearch-fe.de>
```
Comment (ParianGol) [inline analysed-packages/libnice/version-0.1.18/README.md]:
```suggestion
Sascha Dierberg
```

File: analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt [hunk 2]
```diff
@@ -0,0 +1,1506 @@
...
+
+--------------------------------------------------------------------------------
+
+================================================================================
+
+LICENSES 
+
+--------------------------------------------------------------------------------
+
+LicenseRef-Dual-license
```
Comment (ParianGol) [inline analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt]:
```suggestion
Dual-license
```

Comment (ParianGol) [reply analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt]: Two things are important here:

1. There should be no LicenseRef- in the name of licenses in OSS-disclosure file. you can remove it via tool. Please add this line to your python config.yaml file `replacement_regex_txt : [LicenseRef-="", LicenseRef-fossology-=""]`
2. The license text should be something like this https://github.com/Open-Source-Compliance/package-analysis/blob/main/analysed-packages/acl/version-2.3.2/acl-2.3.2-OSS-disclosure.txt#L1508

File: analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt [hunk 3]
```diff
@@ -0,0 +1,1506 @@
...
+
+< signature of Ty Coon > , 1 April 1990
+
+Ty Coon, President of Vice
+
+That's all there is to it!
+
+--------------------------------------------------------------------------------
+
+gary-s-brown
```
Comment (ParianGol) [inline analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt:895]: @dierberg @dks-kraus This license is really new to me. Is this a valid license name?

**Review by ParianGol — COMMENTED** [review]
minor feedbacks! It looks great!

File: analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt (hunk 3 above)
Comment (dierberg) [reply analysed-packages/libnice/version-0.1.18/libnice-0.1.18-OSS-disclosure.txt:895]: Yes, found by scancode

**Comment by dierberg** [comment]
@ParianGol what do we do with repeated Copyrights? They are all in the files, its not a Fossy bug or so.

**Comment by ParianGol** [comment]
> @ParianGol what do we do with repeated Copyrights? They are all in the files, its not a Fossy bug or so.

Yes, I think so. In my case, I was using reuse functionality of fossology. I have noticed in reuse functionality, it had user finding for copyright activated. So user finding were the findings that they were imported and once again your finding so it was stated twice. I can show you!  Although I am not sure that is also the case here ...

**Comment by dierberg** [comment]
No, it is not the reuse bug. there a some files collected in one so we got many copyright entries.

**Pull request merged**